src/Tools/dotnet-watch/test/TestProjects/GlobbingApp/GlobbingApp.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Use `$(DefaultNetCoreTargetFramework)` in all of the `dotnet-watch` tests, not just ../../dotnet-watch.Tests.csproj (no Directory.Build.props file messes that up)

Reply:
During the test, an empty Directory.Build.props and Directory.Build.Targets is generated that prevents us from being able to use `$(DefaultNetCoreTargetFramework)`